src/PortingAssistantExtensionClientShared/Common/UserSettings.cs

```diff
@@ -36,7 +36,9 @@ public sealed class UserSettings
 
         private UserSettings()
         {
+#if Dev16
```

Comment:
this seems concerning, why we need to remove this check?

Reply:
so the execution won't happen on UI thread? that will make us lose the UI context